includes/media_bynder.utils.inc

```diff
@@ -27,27 +27,48 @@ function media_bynder_rest_client() {
  * @todo add support for type and effect
  */
 function media_bynder_embed_url($fragment_id, $type=false, $effects=false, $name = null){
-    $current_media = variable_get('current_media');
+    if(!$type){
+        $type = "web";
+    }
+    $base_url = variable_get('media_bynder_url', '');
+    if (substr($base_url, -1) != '/') {
+        $base_url .= '/';
+    }
+    $base_url .= 'm/' . $fragment_id . '/' . $type . '-drupal.jpg';
```

Comment:
you should verify ```$fragment_id``` and ```$type``` contain the expected content, right now you can call this function with random arguments and return a broken base_url, with for example path traversal in it.